binheap/binheap.py

```diff
@@ -0,0 +1,90 @@
+#! /usr/bin/env python
+# Queue Python file for Efrain, Mark and Henry
+
+
+class Binheap(object):
+    def __init__(self):
+        self.items = []
+
+    def push(self, value):
+        """Add value to bottom of binheap."""
+        self.items.append(value)
+        if len(self.items) > 1:
+            self.sort_bottom()
+
+    def pop(self):
+        """Remove top value from the binheap."""
+        top = self.items[0]
+        self.sort_top()
+        return top
+
+    def parent(self, position):
+        """Return parent of current position."""
+        if position == 0:
+            raise AttributeError("At top of tree.")
+        return (position-1)/2
```

Comment:
be careful here.  In python 3 this will return non-integers for even values of `position` because true division works right in Python 3.  If you want floored division, which you do, you should use the `//` operator instead.